<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageMaterial</class>
 <widget class="QMainWindow" name="ManageMaterial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>质控品维护</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icon.qrc">
    <normaloff>:/images/clipboard.png</normaloff>:/images/clipboard.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="inputKeyword"/>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget_1" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>80</height>
             </size>
            </property>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>330</x>
               <y>40</y>
               <width>54</width>
               <height>12</height>
              </rect>
             </property>
             <property name="text">
              <string>产地</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>330</x>
               <y>10</y>
               <width>54</width>
               <height>12</height>
              </rect>
             </property>
             <property name="text">
              <string>厂家</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>162</x>
               <y>44</y>
               <width>54</width>
               <height>12</height>
              </rect>
             </property>
             <property name="text">
              <string>亚专业</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>71</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>名称</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>11</x>
               <y>45</y>
               <width>54</width>
               <height>12</height>
              </rect>
             </property>
             <property name="text">
              <string>专业</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="inputCountry">
             <property name="geometry">
              <rect>
               <x>360</x>
               <y>40</y>
               <width>113</width>
               <height>20</height>
              </rect>
             </property>
            </widget>
            <widget class="QComboBox" name="inputCategory">
             <property name="geometry">
              <rect>
               <x>200</x>
               <y>40</y>
               <width>111</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QComboBox" name="inputSubject">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>40</y>
               <width>111</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QLineEdit" name="inputManufacturer">
             <property name="geometry">
              <rect>
               <x>360</x>
               <y>10</y>
               <width>113</width>
               <height>20</height>
              </rect>
             </property>
            </widget>
            <widget class="QLineEdit" name="inputName">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>10</y>
               <width>271</width>
               <height>20</height>
              </rect>
             </property>
            </widget>
            <widget class="QPushButton" name="btnSearch">
             <property name="geometry">
              <rect>
               <x>520</x>
               <y>10</y>
               <width>51</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QTableWidget" name="table"/>
             </item>
             <item>
              <widget class="QWidget" name="widget_4" native="true">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <widget class="QPushButton" name="btnAdd">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>360</y>
                  <width>75</width>
                  <height>23</height>
                 </rect>
                </property>
                <property name="text">
                 <string>添加</string>
                </property>
               </widget>
               <widget class="QPushButton" name="btnClose">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>390</y>
                  <width>75</width>
                  <height>23</height>
                 </rect>
                </property>
                <property name="text">
                 <string>关闭</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>inputKeyword</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>inputCountry</tabstop>
  <tabstop>inputManufacturer</tabstop>
  <tabstop>inputName</tabstop>
  <tabstop>inputSubject</tabstop>
  <tabstop>inputCategory</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>table</tabstop>
 </tabstops>
 <resources>
  <include location="../Icon.qrc"/>
 </resources>
 <connections/>
</ui>
